src/Console/Locale/EnableLocaleCommand.php

```diff
@@ -237,7 +237,7 @@ private function askToInstall(): bool
         $this->installWorkingLocale();
 
         $this->formatter->writeln('<info>Copying pages from the default locale to the current locale</info>');
-        BackendPagesModel::copy($this->defaultEnabledLocale, $this->workingLocale);
+        BackendPagesModel::copy($this->defaultEnabledLocale, $this->workingLocale, null);
```

Comment:
if you change the typehint to `Page = null` instead of `?Page` you won't need to do this

Reply:
Changed it to have `copyOne` and `copyAll` methods. 184ffee2